File: C#/FizzBuzzOneLiner.cs
// FizzBuzz LINQ One Liner
// Author: @bennybackslash

using System;
using System.Linq;

namespace FizzBuzz
{
    class Fizzy
    {
        static void Main(string[] args)
        {
            Enumerable.Range(1, 100).Select(i =>(i % 3 == 0 && i % 5 == 0) ? "FizzBuzz" : (i % 3 == 0) ? "Fizz" : (i % 5 == 0) ? "Buzz" : i.ToString()).ToList().ForEach(Console.WriteLine);
        }
    }
}

File: Julia/fizzbuzz-oneliner.jl
#!/usr/bin/julia
# FizzBuzz in one line
# Author: @joaogui1

map(println, collect(i % 15 == 0 ? "FizzBuzz" : i % 5 == 0 ? "Buzz" : i % 3 == 0 ? "Fizz" : i for i in 1:100))
